### PR TITLE
Update Documentations, including tag limit

### LINKED ISSUE #245: Duplicate legend in PPP
image

The above legends seems to be be appearing again under section, "How to use this guide" in your PPP. May be remove the one under the section, "My role"? Since those legends are not used until "How to use this guide".

### PR DESCRIPTION
**Pull request by choongyx**
Update DG DevOps, Testing and Documentation sections.
Minor updates to UG Quick Start section and change tag limit.

Update stats and graph section for UG and DG.
Update PPP.
Remove unused code and tests.

Fixes #245

### CODE REVIEW

File: docs/DeveloperGuide.adoc [hunk 1]
```diff
@@ -915,24 +984,192 @@ Violates Single Responsibility Principle and Separation of Concerns as `HistoryM
...
+=== Types of tests
+
+We have three types of tests:
+
+.  _Unit tests_ targeting the lowest level methods/classes. +
+e.g. `seedu.address.commons.StringUtilTest`
+.  _Integration tests_ that are checking the integration of multiple code units (those code units are assumed to be working). +
+e.g. `seedu.address.storage.StorageManagerTest`
+.  Hybrids of unit and integration tests. These test are checking multiple code units as well as how the are connected together. +
+e.g. `seedu.address.logic.LogicManagerTest`
```
Comment (Nanosync) [inline docs/DeveloperGuide.adoc]: seedu.moneygowhere

File: docs/DeveloperGuide.adoc [hunk 2]
```diff
@@ -915,24 +984,192 @@ Violates Single Responsibility Principle and Separation of Concerns as `HistoryM
...
+
+=== Documentation Previews
+
+When a pull request has changes to asciidoc files, you can use https://www.netlify.com/[Netlify] to see a preview of how the HTML version of those asciidoc files will look like when the pull request is merged. See <<UsingNetlify#, UsingNetlify.adoc>> for more details.
+
+=== Making a Release
+
+Here are the steps to create a new release.
+
+.  Update the version number in link:{repoURL}/src/main/java/seedu/address/MainApp.java[`MainApp.java`].
```
Comment (Nanosync) [inline docs/DeveloperGuide.adoc]: seedu/moneygowhere/

**Review by Nanosync — CHANGES_REQUESTED** [review]
Looks good! Just need to address minor changes

File: docs/DeveloperGuide.adoc (hunk 1 above)
Comment (choongyx) [reply docs/DeveloperGuide.adoc]: Done!

**Review by Nanosync — APPROVED** [review]

**Pull request merged**